server/start.go

```diff
@@ -45,6 +46,7 @@ func StartCmd(ctx *Context, appCreator AppCreator) *cobra.Command {
 	cmd.Flags().String(flagAddress, "tcp://0.0.0.0:26658", "Listen address")
 	cmd.Flags().String(flagTraceStore, "", "Enable KVStore tracing to an output file")
 	cmd.Flags().String(flagPruning, "syncable", "Pruning strategy: syncable, nothing, everything")
+	cmd.Flags().String(flagMinimumFees, "", "Minimum fees for ante handler spam prevention")
```

Comment:
Think this could change a bit? Maybe `Minimum fees validators will accept for a transaction`?

Reply:
s/validators/validator/ - this is a local config that only applies to the local node. ACKd otherwise.